generators/src/main/java/com/pholser/junit/quickcheck/generator/java/time/DayOfWeekGenerator.java

```diff
@@ -0,0 +1,49 @@
+/*
+ The MIT License
+
+ Copyright (c) 2010-2015 Paul R. Holser, Jr.
+
+ Permission is hereby granted, free of charge, to any person obtaining
+ a copy of this software and associated documentation files (the
+ "Software"), to deal in the Software without restriction, including
+ without limitation the rights to use, copy, modify, merge, publish,
+ distribute, sublicense, and/or sell copies of the Software, and to
+ permit persons to whom the Software is furnished to do so, subject to
+ the following conditions:
+
+ The above copyright notice and this permission notice shall be
+ included in all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+package com.pholser.junit.quickcheck.generator.java.time;
+
+import com.pholser.junit.quickcheck.generator.GenerationStatus;
+import com.pholser.junit.quickcheck.generator.Generator;
+import com.pholser.junit.quickcheck.random.SourceOfRandomness;
+
+import java.time.DayOfWeek;
+
+import static com.pholser.junit.quickcheck.internal.Items.choose;
+import static java.util.Arrays.asList;
+
+/**
+ * Produces values of type {@link DayOfWeek}.
+ */
```

Comment:
Unless used with `From`, custom generators for `enum` types will be trumped by the built-in `EnumGenerator`. I think we can eliminate the `DayOfWeekGenerator` and `MonthGenerator`. It might be good to have tests that demonstrate that you can have these `enum`s as property parameters, though.